studio-docs/source/org/organizations.mdx

```diff
@@ -5,8 +5,6 @@ sidebar_title: Organizations
 
 All data in Apollo Studio (GraphQL schemas, metrics, etc.) belongs to a particular organization. Every organization has one or more [members](./members/) who manage it and can access its associated data.
 
-> **Important**: Currently, all members of a Studio organization have full permissions for the organization, including the ability to delete graphs or transfer them out of the organization.
```

Comment:
I think this is explained enough on the members page and that we don't need to mention it here

Reply:
I'd prefer to keep the notice until the feature is GA because it's an important security consideration and because it now allows us to link to the info about roles in another spot (so people can't look in the wrong place)